IMS.BL/InventoryRepository.cs

```diff
@@ -1,21 +1,31 @@
 using System;
 using System.Collections.Generic;
+using System.Data.SqlClient;
 using System.Linq;
+using IMS.BL.DataService;
+using IMS.DL.SqlDatabaseConnection;
 
 namespace IMS.BL
 {
     public class InventoryRepository
```

Comment:
who Inventory and InventoryRepository difference from each other, they are both the repositories of inventory, you have to do a system wiled refactoring, that is the task.